config/default.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
-  port: process.env.PORT || 3000,
+  port: process.env.PORT || 9200,
+  loggedInRedirectUrl: 'http://dev.everreal.co:9300/properties/decision',
```

Comment:
This link will decide where to redirect the user next, depending on his registration state. Eg: after registration, did he finished setting up his account or not - like giving us all the information we initially require when he is owner, property manager, tenant, etc.